packages/protoplugin-test/src/transpile.test.ts

```diff
@@ -107,120 +102,6 @@ describe("transpile", function () {
     ]);
   });
 
-  describe("file preamble", () => {
```

Comment:
Not sure why those tests where here. Moved to a different file and increased coverage.